orbital-mechanics/Lecture7/Lecture7.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>I'm not sure why it's display $Problem$ again. Maybe I missed it the first time when correcting. But when I go onto the preview through github.com, this is not a rendering issue. Same with the similar issues below. They don't show up on the github.com preview. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Well, one thing you can do is get rid of the "$$ $$"; that is for inline equations. Here you are just writing a word. What you can do is just start a new line and write {\bf Problem}.</p><p> </p>